catalog-data.ttl

```diff
@@ -4151,3 +4154,12 @@ cdata:Zhilin_Zhang a ex:Person ;
 cdata:Zongyan_Li a ex:Person ;
   ex:name "Zongyan Li" ;
   ex:subType con:Person .
+
+<urn:uuid:3b05c17c-f44a-470e-8031-408abee6e2ca> a ex:Person ;
```

Comment:
@ericprud what is your WebId? Are there other handles you'd like to add whilst you're here?